pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -247,3 +256,122 @@ def convert_nested_indexer(indexer_type, keys):
             index=MultiIndex.from_product(keys))
 
         tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize('indexer, is_level1, expected_error', [
+    ([], False, None),  # empty ok
+    (['A'], False, None),
+    (['A', 'D'], False, None),
+    (['D'], False, r"\['D'\] not in index"),  # not any values found
+    (pd.IndexSlice[:, ['foo']], True, None),
+    (pd.IndexSlice[:, ['foo', 'bah']], True, None)
+])
+def test_getitem_duplicates_multiindex_missing_indexers(indexer, is_level1,
```

Comment:
I suppose it's better to rename these `get_loc` since that is in fact what they are testing

Reply:
have renamed tests moved to test_loc.py to include  _loc_getitem_